tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -480,13 +480,13 @@ def test_positive_update_external_user_roles(
         )
     with target_sat.ui_session(
         test_name, ldap_data['ldap_user_name'], ldap_data['ldap_user_passwd']
-    ) as session:
+    ) as ldapsession:
```

Comment:
Probably easier would be just to remove the `session` fixture from the test signature - it's deprecated anyway - and update the line 459 above to `with target_sat.ui_session() as session:`.
Then you don't need two different variable names for each session and you won't have the name conflict with the `session` variable/fixture either.

Reply:
But I need multiple sessions here anyway, it's intentional.
About changing deprecated `session`, I could do that but I'm currently just trying to fix failing tests to make automation greener. Not really a time for refactoring. There would surely be multiple instances of this and it's better to fix them in one commit.

Reply:
Ok, as a simple test fix, I can live with it. Indeed there is still plenty of occurrences of the old `session` fixture.